src/form-handlers/handler/checkbox.handler.js

```diff
@@ -15,20 +15,18 @@ class CheckboxHandler {
   }
 
   handleFill(page, elementName, desiredValue) {
-    return page[elementName].filter(function (elem) {
-      return elem.element(by.xpath('..')).getText().then(function (text) {
-        return text === desiredValue;
+    return page[elementName].filter(elem => {
+      return elem.element(by.xpath('..')).getText().then((text) => {
+        return text.trim() === desiredValue;
```

Comment:
For example, there is an element with value:
```
      Checkbox Option 3
      ```
`Google Chrome` and `Firefox` length is equal to 17 but in Safari it returns `43`. So the trim operation is done by default in those browsers.

Reply:
Nice catch!